### PR TITLE
Forgetpassword correction and some signup page fixes

### PR DESCRIPTION
**Pull request by Devilla**

### CODE REVIEW

File: src/components/WebsiteSignUp/WebsiteSignUp.js [hunk 1]
```diff
@@ -131,19 +142,22 @@ class WebsiteSignUp extends Component {
...
                       <strong>{error}</strong>
                       </Alert>
                     }
-                    <hr className="short" />
+                    <div className="mt-5"  style={{backgroundColor:"#f8d0c8" ,fontSize:"15px"}} id="av-flash-block" className="c-flashBlock" >   <div data-mc-group="flashBlock"  c > <ul className="pt-2 pb-2"><li>Please check your entry and try again.</li></ul> </div> </div>
+                    <hr />
+                    
                     <form>
-                      <div className="row">
+                      <div classNameName="row">
```
Comment (ShankyRana) [inline src/components/WebsiteSignUp/WebsiteSignUp.js]: correct this.

File: src/components/WebsiteSignUp/WebsiteSignUp.scss [hunk 2]
```diff
@@ -1,10 +1,204 @@
...
+    float: left;
 }
 
 .alert-warning {
   background-color: #f2dede;
   border-color: #ebccd1;
   color: #a94442;
 }
+
+.\!padding-top-bottom--lv0 {
```
Comment (ShankyRana) [inline src/components/WebsiteSignUp/WebsiteSignUp.scss]: \!  ?

File: src/components/WebsiteSignUp/WebsiteSignUp.scss [hunk 3]
```diff
@@ -1,10 +1,204 @@
...
+  padding-top: 0 !important;
+  padding-bottom: 0 !important;
+}
+.padding--lv3 {
+  padding: 18px;
+}
+
+
+element.style {
```
Comment (ShankyRana) [inline src/components/WebsiteSignUp/WebsiteSignUp.scss]: remove this

File: src/components/WebsiteSignUp/WebsiteSignUp.scss [hunk 4]
```diff
@@ -1,10 +1,204 @@
...
+.v-flashBlock--error {
+    background-color: #f8d0c8;
+}
+.margin-bottom--lv3 {
+  margin-bottom: 18px !important;
+}
+.av-flash {
+    margin-top: 30px;
+}
+html, body, div, span, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, code, del, dfn, em, img, q, dl, dt, dd, ol, ul, fieldset, form, label, legend, textarea, table, caption, tbody, tfoot, thead, tr, th, td {
```
Comment (ShankyRana) [inline src/components/WebsiteSignUp/WebsiteSignUp.scss]: css is repeated please remove it.

File: src/components/WebsiteSignUp/WebsiteSignUp.scss [hunk 5]
```diff
@@ -31,9 +31,6 @@
 .margin-bottom--lv3 {
   margin-bottom: 18px !important;
 }
-.av-flash {
-    margin-top: 30px;
-}
 html, body, div, span, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, code, del, dfn, em, img, q, dl, dt, dd, ol, ul, fieldset, form, label, legend, textarea, table, caption, tbody, tfoot, thead, tr, th, td {
```
[the remark this answers is not in the document]
Comment (Devilla) [reply src/components/WebsiteSignUp/WebsiteSignUp.scss]: It's removal is ok. Going to remove in linting too. Can we have "circle-tick" glyphicon like mailchimp at start of blue strip at the bottom.

**Pull request closed without merging**